crates/bevy_winit/src/lib.rs

```diff
@@ -94,10 +94,14 @@ fn change_window(_: &mut World, resources: &mut Resources) {
                     let window = winit_windows.get_window(id).unwrap();
                     window.set_cursor_visible(visible);
                 }
-                bevy_window::WindowCommand::SetCursorPosition { x, y } => {
+                bevy_window::WindowCommand::SetCursorPosition { position } => {
                     let window = winit_windows.get_window(id).unwrap();
+                    let inner_size = window.inner_size().to_logical::<f32>(window.scale_factor());
```

Comment:
You might be able to get rid of the type hint here since you have subtraction below assigning to an f32, which should hint the compiler enough to figure it out, I haven't tested that but it feels like it would be the case.

Reply:
I tried but the compiler can't figure it out.

Reply:
I'm also very cool with explicit types in case like this, even if they can be inferred. Helps convey intent / protects against implict behavior changes when types change

Reply:
I am disappointed in the compiler ):